src/vcf/assembly_checker.cpp

```diff
@@ -74,6 +107,107 @@ namespace ebi
           return outputs[0]->is_valid_report();
       }
 
+      void create_uncompressed_stream(std::istream & input,
```

Comment:
did you copy all this code after deciding it's not worth to extract, or are you working on extracting it?